management/persistence.md

```diff
@@ -0,0 +1,330 @@
+* RDB needs to fork() often in order to persist on disk using a child process. fork() can be time consuming if the dataset is big, and may result in FalkorDB stopping serving clients for some milliseconds or even for one second if the dataset is very big and the CPU performance is not great. AOF also needs to fork() but less frequently and you can tune how often you want to rewrite your logs without any trade-off on durability.
+
+## AOF advantages
+
+* Using AOF FalkorDB is much more durable: you can have different fsync policies: no fsync at all, fsync every second, fsync at every query. With the default policy of fsync every second, write performance is still great. fsync is performed using a background thread and the main thread will try hard to perform writes when no fsync is in progress, so you can only lose one second worth of writes.
+* The AOF log is an append-only log, so there are no seeks, nor corruption problems if there is a power outage. Even if the log ends with a half-written command for some reason (disk full or other reasons) the FalkorDB-check-aof tool is able to fix it easily.
+* FalkorDB is able to automatically rewrite the AOF in background when it gets too big. The rewrite is completely safe as while FalkorDB continues appending to the old file, a completely new one is produced with the minimal set of operations needed to create the current data set, and once this second file is ready FalkorDB switches the two and starts appending to the new one.
+* AOF contains a log of all the operations one after the other in an easy to understand and parse format. You can even easily export an AOF file. For instance even if you've accidentally flushed everything using the `FLUSHALL` command, as long as no rewrite of the log was performed in the meantime, you can still save your data set just by stopping the server, removing the latest command, and restarting FalkorDB again.
+
+## AOF disadvantages
+
+* AOF files are usually bigger than the equivalent RDB files for the same dataset.
+* AOF can be slower than RDB depending on the exact fsync policy. In general with fsync set to *every second* performance is still very high, and with fsync disabled it should be exactly as fast as RDB even under high load. Still RDB is able to provide more guarantees about the maximum latency even in the case of a huge write load.
+
+
+Ok, so what should I use?
+---
+
+The general indication you should use both persistence methods is if
+you want a degree of data safety comparable to what PostgreSQL can provide you.
+
+If you care a lot about your data, but still can live with a few minutes of
+data loss in case of disasters, you can simply use RDB alone.
+
+There are many users using AOF alone, but we discourage it since to have an
+RDB snapshot from time to time is a great idea for doing database backups,
+for faster restarts, and in the event of bugs in the AOF engine.
```

Comment:
If you can't afford data lose... (this is the AOF on every write case)
don't mention bugs in the AOF engine.